domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/MeasurementsSyncServiceTest.cs

```diff
@@ -67,7 +67,7 @@ public async Task FetchMeasurements_MeasurementsReceived_SyncPositionUpdated()
             {
                 new Measurement
                 {
-                    Gsrn = syncInfo.GSRN,
+                    Gsrn = syncInfo.Gsrn.Value,
```

Comment:
This GSRN should also be GSRN. Value objects can be mapped to db entries. So no need to make it as .Value

Reply:
Measurement here is from gRPC proto file, so can't do much about that